py-polars/polars/datatypes/classes.py

```diff
@@ -4,47 +4,49 @@
 from inspect import isclass
 from typing import TYPE_CHECKING, Any, Iterator, Mapping, Sequence
 
-import polars.datatypes
+import polars.datatypes as dt
```

Comment:
Could this be imported with a different name? `dt` reminds me of datetime.

Reply:
Fair enough, we could do `pdt`.